src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -104,6 +105,18 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         /// </summary>
         public string? RuntimeMetadataVersion { get; private set; }
 
+        /// <summary>
+        /// The default source encoding used to parse source before producing a compilation. Only embedded in
+        /// the pdb if the value is non-null. 
+        /// </summary>
+        public Encoding? DefaultSourceFileEncoding { get; private set; }
+
+        /// <summary>
+        /// The fallback source encoding used to parse source before producing a compilation. Only embedded
```

Comment:
>compilation [](start = 81, length = 11)

"If <see cref="DefaultSourceFileEncoding"/> is not specified, the encoding used to parse source files that do not declare their encoding via Byte Order Mark (BOM) and are not UTF8-encoded."